src/webview/columns/data-column.tsx

```diff
@@ -31,67 +55,240 @@ export class DataColumn implements ColumnContribution {
     readonly label = 'Data';
     readonly priority = 1;
 
-    protected byteGroupStyle: React.CSSProperties = {
-        marginRight: `${DataColumn.Styles.MARGIN_RIGHT_PX}px`
+    render(range: BigIntMemoryRange, memory: Memory, options: TableRenderOptions): React.ReactNode {
+        const byteGroups = this.parse({ memory, range, renderOptions: options });
+        return byteGroups.map(group =>
+            <EditableDataColumnGroup key={group.startAddress.toString(16)} range={range} group={group} options={options} writeMemory={this.writeMemory}></EditableDataColumnGroup>);
+    }
+
+    protected writeMemory = async (writeArguments: DebugProtocol.WriteMemoryArguments): Promise<void> => {
+        await messenger.sendRequest(writeMemoryType, HOST_EXTENSION, writeArguments);
```

Comment:
In the webview code, please use the new types that @martin-fleck-at has introduced in `messsaging.ts` rather than `DebugProtocol` types. Since this function doesn't refer to any state kept by the class, it could be moved down into `EditableDataColumnGroup` or out into module scope, but it can also be left here i you think it might later refer to some state on this class.